src/Apps/Order/Components/TransactionDetailsSummaryItem.tsx

```diff
@@ -162,6 +169,43 @@ export const TransactionDetailsSummaryItem: FC<TransactionDetailsSummaryItemProp
     }
   }
 
+  const taxPrefix = () => {
+    let prefix
+    if (offerShippingCostSubjectToChange()) {
+      prefix = <Sup>†</Sup>
+    } else {
+      prefix = "*"
+    }
+    return <>{prefix}Additional duties and taxes </>
+  }
+
+  const taxLabel = () => {
+    let suffix
+    if (offerShippingCostSubjectToChange()) {
+      suffix = <Sup>†</Sup>
+    } else {
+      suffix = "*"
+    }
+    return <>Tax{suffix}</>
+  }
+
+  const offerShippingCostSubjectToChange = () => {
+    const offer = getOffer()
+
+    return (
+      offer?.shippingTotalCents &&
+      order.requestedFulfillment?.__typename === "CommerceShipArta" &&
+      offerStillInNegotiation()
+    )
+  }
+
+  const offerStillInNegotiation = () => {
+    return (
+      order.mode === "OFFER" &&
+      ["PENDING", "SUBMITTED"].includes(order.displayState)
```

Comment:
DisplayState check needed to not show this message on the `Details` view in conversions after everything has been approved

This screen: 
<img width="320" alt="Screen Shot 2023-05-25 at 5 32 11 PM" src="https://github.com/artsy/force/assets/12748344/e19406c0-c874-4d00-acfe-1c443f7cdd02">

Reply:
This screen:
<img width="1680" alt="Screen Shot 2023-05-25 at 5 47 48 PM" src="https://github.com/artsy/force/assets/12748344/c5d7b0ec-7456-4ee1-9ed8-19fa0d3009a7">